.env.example

```
@@ -15,6 +15,10 @@ FC_DB_SQLITE_PATH=./db
 # Cache Configuration
 FC_REDIS_URI=redis://localhost:6379/
 
+# Outbound HTTP User-Agent defaults
+FC_HTTP_USER_AGENT_FEED=FeedCraft/2.0
+FC_HTTP_USER_AGENT_HTML=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)